src/Scaffold_Command.php

```diff
@@ -902,15 +902,18 @@ private function pluralize( $word ) {
 			'/$/'                      => 's'
 		);
 
-		$uncountable = array( 'equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep' );
+		$uncountable = array( 'sheep', 'fish', 'deer', 'series', 'species', 'money', 'rice', 'information', 'equipment' );
```

Comment:
@Umangvaghela what @miya0001 meant is that don't change the order of the default array. If you want to add new elements then add it to the end of the array.